### DISCUSSION TITLE
Cloud Optimized GeoTiff

### DISCUSSION THREAD

**Discussion by imakihi** [TerriaJS/terriajs#5890] [Ideas] [answered]
Hi there.
I just thought it would be really nice if I can use COG (https://www.cogeo.org/) as an catalog item to show my raster data.

Just an idea.

Best.

Hiroo

**Comment by nf-s** [comment, marked as answer]
Hi @imakihi 

We currently don't have short-term plans to add Geotiff/COG support - but I would love to add support for it. 
Especially with all of the new STAC stuff coming out!

Cheers,
Nick

**Comment by imakihi** [comment]
Nick. Thank you for your comment. It would be super cool to see if we can
use COG sometime.
Thank you!
Hiroo

On Tue, Oct 5, 2021 at 10:25 AM Nick Forbes-Smith ***@***.***>
wrote:

> Hi @imakihi <https://github.com/imakihi>
>
> We currently don't have short-term plans to add Geotiff/COG support - but
> I would love to add support for it.
> Especially with all of the new STAC stuff coming out!
>
> Cheers,
> Nick
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/TerriaJS/terriajs/discussions/5890#discussioncomment-1427593>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABEZ25APPVSXEF4ZEDUTTPLUFJHYLANCNFSM5FHLABGQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Hiroo Imaki (今木 洋大)
Pacific Spatial Solutions Inc. ｜ http://www.pacificspatial.com
3-3-1 Marunouchi, Shin-Tokyo Bldg. 264
Chiyoda-ku, Tokyo  100-0005
